### PR TITLE
Add Context to README

### PR DESCRIPTION
**Pull request by laurapacilio**
As I was working on the changelog and other marketing messaging for bootstrap, I thought it might be helpful to add some context to the Bootstrap README. Specifically, I want to propose:
- Updating the introduction to encourage folks to contact us for help setting up a custom integration (rather than trying to go it alone, which we've discussed is not yet our preferred path for new users).
- Giving a brief, high-level overview of the bootstrap read/write workflow and which Stedi products it uses. The folks who use the bootstrap will likely be folks who are very new to Stedi and our products. I think we can use this as an opportunity to help them build a mental model of how our building blocks work and how they fit together into an end-to-end integration. I also think briefly introducing terms like bucket in context here, may help them more quickly understand what's going on later when they follow the instructions to test the system. Hopefully this helps new users out without adding too much bulk to the README :) 
- Adding a short section at the end with next steps to customize the bootstrap. I saw that Joost wrote some great instructions about how to customize, but I realized that users may not necessarily know that those exist/where to find them since they aren't part of this repository. So, I called out a few specific actions (again, to help folks build a mental model) that users can take, and link them to the specific part of our docs that describes how to do those tasks. 

Of course, please push back on anything I wrote that's incorrect (I did my best to infer what's going on, but I may have mischaracterized things!) or that you feel is inappropriate for this context.  Thank you!

### CODE REVIEW

File: README.md
Comment (laurapacilio) [inline README.md:352]:
```suggestion
- [Add a partner profile](https://www.stedi.com/docs/bootstrap/adjusting-the-workflow#add-a-trading-partner-profile) to replace the test trading partner with your real trading partners' details and requirements.
```

File: README.md
Comment (laurapacilio) [inline README.md:3]:
```suggestion
This repository contains an end-to-end configuration for building a full X12 EDI system using Stedi products. This implementation demonstrates one way to build an integration for the common read and write EDI use case. Your solution may differ, depending on your systems and requirements. We encourage you to [contact us](mailto:support@stedi.com) as you get started with Stedi, so we can help you design a custom integration.
```

File: README.md
Comment (joost-basic-bits) [inline README.md:10]: The product is called Stedi Buckets, the resource is called a Stedi bucket. Which reminds me, I need to add that to the style guide. :-)

File: README.md
Comment (joost-basic-bits) [inline README.md:12]: I don’t add _Stedi_ to product that start with EDI. I don’t think we decided on a naming scheme, but that’s what I use at the moment. So, Stedi Buckets, but EDI Translate. I started using the _Stedi_ prefix to avoid confusion (since our products have generic names like Buckets, Functions, SFTP, etc.), but I don’t think that’s necessary for names like EDI Translate and EDI Core; no risk of confusion there.

File: README.md
Comment (joost-basic-bits) [inline README.md:8]: I think we need to come up with a way to refer to the bootstrap repository. I use _bootstrap workflow_ most of the time and only use _bootstrap repository_ if I’m talking about the GitHub repository. I see you use _the bootstrap_. I started out using just _Bootstrap_, but Mohan prefers not treating it like a product, so not capitalization. I’m not saying we have to do it as I did, just that it would be good to pick one way.

File: README.md
Comment (joost-basic-bits) [inline README.md:50]: Is this correct? This puts _Testing the system_ on the same level as _Stedi EDI Bootstrap_. I think it was correct before.

File: README.md
Comment (joost-basic-bits) [inline README.md:347]:
```suggestion
5. You can view the file using the [Buckets UI](https://www.stedi.com/app/buckets). As shown above, the output of the function includes the `bucketName` and `key` (path within the bucket) of where the generated EDI was saved.
```

Comment (joost-basic-bits) [reply README.md:347]: I know you didn’t change this, Laura, but I missed this before. This is another thing we haven’t been able to agree on, but the docs are pretty consistent these days in using _UI_ for the, well, UI.

File: README.md
Comment (joost-basic-bits) [inline README.md:349]: As before: we should decide on how we refer to The Bootstrap.

File: README.md
Comment (joost-basic-bits) [inline README.md:353]:
```suggestion
- [Create a Stedi mapping](https://www.stedi.com/docs/bootstrap/adjusting-the-workflow#map-inbound-messages) that transform EDI documents into a custom JSON shape for your internal system. The base bootstrap repository ingests and generates JSON with a schema that closely matches EDI documents. 
```

Comment (joost-basic-bits) [reply README.md:353]: _Stedi Mappings_ is the product. We don’t want our customers to create that; we’d have nothing left to do! ;-)

File: README.md
Comment (joost-basic-bits) [inline README.md:10]:
```suggestion
1. An example trading partner uses [Stedi SFTP](https://www.stedi.com/products/sftp) to send EDI documents to a [Stedi bucket](https://www.stedi.com/products/buckets).
```

Comment (joost-basic-bits) [reply README.md:10]: The product is called Stedi Buckets, the resource is called a Stedi bucket. Which reminds me, I need to add that to the style guide. :-)

File: README.md
Comment (laurapacilio) [reply README.md:12]: Yeah this is something I keep going back and forth on, so I'm really glad you brought it up. My opinion is that we should always use Stedi in front of the ambiguous ones like you have already been doing, and then be able to use it front of other product names if it's the first reference to the product/if required in context. I realize this may be a point of contention haha, but just saying my thoughts on this. I'm going to look into the way other folks handle this because I want it to be in our editorial style guide. I might leave it in this copy unless you feel VERY strongly about me removing it.

File: README.md
Comment (laurapacilio) [inline README.md:50]:
```suggestion
## Testing the system
```

File: README.md
Comment (laurapacilio) [inline README.md:6]:
```suggestion
## Read and Write Workflow 
```

File: README.md
Comment (laurapacilio) [inline README.md:18]:
```suggestion
## Prerequisites & Deployment
```

File: README.md
Comment (laurapacilio) [inline README.md:52]:
```suggestion
### Inbound EDI
```

File: README.md
Comment (laurapacilio) [inline README.md:349]:
```suggestion
# Customizing the Bootstrap Workflow
```

File: README.md
Comment (laurapacilio) [reply README.md:347]: thank you!! I am always a fan of opportunistic fixes :)

**Comment by laurapacilio** [comment]
@joost-basic-bits - I think I addressed everything you mentioned, so if you feel comfortable, please approve :)  Oh, the only thing is I left Stedi in front of the translate API. Since this is a repository that's removed from our marketing site (where all our products are listed) I like making doubly sure it's clear that it's our product for new folk. But also I can see the reasonable counterargument which is, "I mean...what else would it be though?" so I can take it out before merging if you feel very strongly it shouldn't be there.

File: README.md
Comment (rsioss) [inline README.md:3]: nit: `s/common read and write EDI use case/common read and write EDI use cases/`

also, I'm not sure about this phrasing: "design a custom integration." maybe something like "configure these workflows to suit your needs" (or something along those lines). cc: @joost-basic-bits for suggestions as well 😄

File: README.md
Comment (rsioss) [inline README.md:10]: A trading partner _could_ use SFTP (and in practice that's probably the common workflow), but in the instructions / testing section we have the user just upload the file using the Buckets UI, it might be better to indicate that here (could also add that uploads via SFTP will trigger this process as well)

File: README.md
Comment (rsioss) [inline README.md:15]: nit: it might be good to specifically mention "handle outgoing EDI" here, to align with the description for incoming EDI above.

File: README.md
Comment (rsioss) [inline README.md:352]: nit: there's a contradiction here in `Add` and `replace` (could either be `Edit` and `replace`, or just `Add` and maybe `to represent your real trading...)`

File: README.md
Comment (rsioss) [inline README.md:353]: another nit: may want to generalize the use case here, as mapping can be in either direction between guide JSON and a custom API (not necessarily just EDI-like JSON to custom API, could be the other way around too)

**Review by rsioss — APPROVED** [review]

File: README.md
Comment (joost-basic-bits) [reply README.md:353]: > nit: s/common read and write EDI use case/common read and write EDI use cases/

Depends on whether you see reading and writing EDI as a single use case or as two.

> I'm not sure about this phrasing: "design a custom integration."

We encourage you to [contact us](mailto:support@stedi.com) as you get started with Stedi, so we can help you customize the workflow.

Although, what you wrote, works too, Rob. @Laura, now you have options. :smile:

File: README.md
Comment (joost-basic-bits) [reply README.md:12]: I think I can manage to suppress my outrage over your flaunting of my personal and arbitrary, but clearly superior, convention around naming our products. :wink:

File: README.md
Comment (joost-basic-bits) [reply README.md:3]: > nit: s/common read and write EDI use case/common read and write EDI use cases/

Depends on whether you see reading and writing EDI as a single use case or as two.

> I'm not sure about this phrasing: "design a custom integration."

We encourage you to [contact us](mailto:support@stedi.com) as you get started with Stedi, so we can help you customize the workflow.

Although, what you wrote, works too, Rob. @Laura, now you have options. :smile:

Comment (laurapacilio) [reply README.md:3]: thank you both! :). Design a custom integration is phrasing I think I've seen elsewhere (maybe??) but I really like the suggested edits. I will update to reflect!

Comment (laurapacilio) [inline README.md:3]:
```suggestion
This repository contains an end-to-end configuration for building a full X12 EDI system using Stedi products. This implementation demonstrates one way to build an integration for the common read and write EDI use cases. Your solution may differ, depending on your systems and requirements. We encourage you to [contact us](mailto:support@stedi.com) as you get started with Stedi, so we can help you customize the workflow.
```

Comment (laurapacilio) [reply README.md:3]: also - good call on making use cases plural!

File: README.md
Comment (laurapacilio) [reply README.md:10]: mm okay that's a good point. Let me adjust!

File: README.md
Comment (laurapacilio) [reply README.md:12]: hahaha Joost you're awesome

File: README.md
Comment (laurapacilio) [reply README.md:15]: excellent call - let me adjust accordingly!

File: README.md
Comment (laurapacilio) [inline README.md:15]:
```suggestion
The bootstrap uses a similar process in reverse to handle outgoing EDI. It translates JSON documents to EDI and publishes them to a Stedi bucket.
```

File: README.md
Comment (laurapacilio) [inline README.md:352]:
```suggestion
- [Edit the partner profile](https://www.stedi.com/docs/bootstrap/adjusting-the-workflow#add-a-trading-partner-profile) to replace the test trading partner with your real trading partners' details and requirements.
```

File: README.md
Comment (laurapacilio) [inline README.md:353]:
```suggestion
- [Create a Stedi mapping](https://www.stedi.com/docs/bootstrap/adjusting-the-workflow#map-inbound-messages). The base bootstrap repository ingests and generates JSON with a schema that closely matches EDI documents. You may need to create a mapping to transform EDI documents into a custom JSON shape for your internal system. You can also create a mapping that transforms JSON data from your system into the JSON schema required for outgoing EDI documents.
```

File: README.md
Comment (laurapacilio) [inline README.md:353]:
```suggestion
- [Create Stedi mappings](https://www.stedi.com/docs/bootstrap/adjusting-the-workflow#map-inbound-messages). The base bootstrap repository ingests and generates JSON with a schema that closely matches EDI documents. You may need to create a mapping to transform EDI documents into a custom JSON shape for your internal system. You can also create a mapping that transforms JSON data from your system into the JSON schema required for outgoing EDI documents.
```

**Pull request merged**
